lib/ramble/ramble/pipeline.py

```diff
@@ -159,6 +168,15 @@ def run(self):
         with self.workspace.logger(self.log_path, echo=self.workspace.dry_run):
             self._complete()
 
+    def simlink_log(self, base, link):
+        """
+        Create simlink of log file to give a known and predictable path
+        """
+        if os.path.islink(link):
+            os.unlink(link)
```

Comment:
I don't know if we do this elsewhere, but should we catch a case where `link` is created, and is not a link? Or should we just let the `symlink` call below fail?

Reply:
What we decided last time was to let the simlink fail to avoid clobbering / silently giving confusing behavior

Reply:
OK cool. Thanks!